ui-backend/intrigue-ui-app/src/main/resources/features.xml

```diff
@@ -88,7 +88,7 @@
 
     <feature name="javalin" version="${project.version}" description="Javalin Web Framework">
         <feature prerequisite="true">wrap</feature>
-        <bundle>wrap:mvn:org.checkerframework/checker-qual/2.0.0$Bundle-Name=checker-qual&amp;Bundle-SymbolicName=checker-qual&amp;Bundle-Version=2.0.0&amp;Export-Package=org.checkerframework.checker.nullness.qual;version="2.0.0"</bundle>
+<!--        <bundle>wrap:mvn:org.checkerframework/checker-qual/2.0.0$Bundle-Name=checker-qual&amp;Bundle-SymbolicName=checker-qual&amp;Bundle-Version=2.0.0&amp;Export-Package=org.checkerframework.checker.nullness.qual;version="2.0.0"</bundle>-->
```

Comment:
❓ Can be deleted?

Reply:
This always gives me trouble when building. That being said there's a chance of runtime issues if this is excluded. I haven't tested enough to confirm it can be removed. It also doesn't seem to give anyone else trouble. So I think I'll leave it in for now, but if it starts causing build issues for anyone else I can revisit removing it.